lib/protocon/plugins/driver_tcp.py

```diff
@@ -93,9 +97,16 @@ def open(self):
 
 		tcp_sock = socket.socket(self._addrinfo.family, self._addrinfo.type)
 		if self.settings['type'] == 'client':
+			if self.url.scheme.startswith('ssl'):
```

Comment:
Brain fart and forgot about `startswith`, despite having used it all past week with `pwntools`. Fixed now.